cl/scrapers/tests.py

```diff
@@ -124,22 +124,21 @@ def test_juriscraper_object_creation(self):
             msg="Expected citation was not created in db",
         )
 
-    def test_juriscraper_object_creation(self):
-        """Can we extract text from tax court pdf and add to db?"""
-
-        test_op = Opinion.objects.get(pk=76)
-        assert (
-            test_op.cluster.citations.exists() is False
-        ), "Citation already exists"
+    def test_juriscraper_docket_number_extraction(self):
+        """Can we extract docket number from tax court pdf and add to db?"""
 
-        extract_doc_content(pk=test_op.pk, do_ocr=False)
-
-        assert (
-            test_op.cluster.citations.exists() is True
-        ), "Citation extraction failed"
-        print(
-            "Successful tax court citation parsing from pdf",
-            test_op.cluster.citation_string,
+        o = Opinion.objects.get(pk=76)
+        self.assertEqual(
+            None,
+            o.cluster.docket.docket_number,
+            msg="Docket number should be none.",
+        )
+        extract_doc_content(pk=76, do_ocr=False)
+        o = Opinion.objects.get(pk=76)
+        self.assertEqual(
+            "19031-13, 27735-13, 11905-14",
+            o.cluster.docket.docket_number,
+            msg="Docket number extracted properly",
```

Comment:
I had a duplicated test - I assume during a rebase.  I replaced the duped test to test docket number extraction.  Tests fail currently, but with the PR juriscraper this test succeeds.

Reply:
This has been fixed and all tests are now passing with the docker image being pushed